reglemente.tex

```diff
@@ -507,6 +526,7 @@ \section{Informationsutskottet}
 		\item Filmare
         \item Redaktör
 		\item Funktionärer
+		\item Webmaster
```

Comment:
Alla andra utskott har Funktionär sist i listan. Kanske najs att behålla det så.

Reply:
Har comittat, bör squahas in men vill ej rebasea nu

Reply:
LGTM, kan du pusha det?